### PR TITLE
Add function to get channel infos using the device id

### PR DESCRIPTION
**Pull request by EndeNeu**
@deeplook please review.

### CODE REVIEW

File: relayr/api.py [hunk 1]
```diff
@@ -1082,8 +1082,20 @@ def delete_channel_batch(self, deviceID = None, transport = None):
             data['transport'] = transport
         _, res = self.perform_request('DELETE', url,
                                       data = data, headers = self.headers)
-            
-    ## TODO: remove in version 0.3.0
+
+    def get_channels_by_device(self, deviceID):
```
Comment (deeplook) [inline relayr/api.py]: Would you mind calling this ``get_device_channels()`` to comply more with the naming conventions used so far?

File: relayr/api.py [hunk 2]
```diff
@@ -1082,8 +1082,20 @@ def delete_channel_batch(self, deviceID = None, transport = None):
...
+        """
+        Get all the channels for a deviceID
+
+        :param deviceID: the device UUID
+        :type deviceID: string
+        :rtype: the deviceId and a list of lists, channelID, appID and transport
```
Comment (deeplook) [inline relayr/api.py]: Could you given an example like in ``post_channel()``?

**Comment by EndeNeu** [comment]
@deeplook done, please review.

**Comment by deeplook** [comment]
Merged manually with eafb9ff53.

**Pull request closed without merging**